_infra/helm/collection-instrument/values.yaml

```diff
@@ -58,6 +58,14 @@ gcp:
     prefix: ""
   topic: topic
 
+gcpDev:
```

Comment:
This name is pretty bad.  Also the config is laid out wrong.  You can remove the `project` as it's the same between them.
The `bucket` should be renamed to `collectionInstrumentBucket` or something like it, should be in the `gcp` block above